submissions/RomanSaiko/oop/index.js

```diff
@@ -0,0 +1,72 @@
+
+class Inhabitant {
+    constructor ({ species, name, gender, saying, friends = "No friends! One man army.", paws, hands, legs }) {
+        this.species = species
+        this.name = name
+        this.gender = gender
+        this.saying = saying
+        this.friends = friends
+        this.paws = paws
+        this.hands = hands
+        this.legs = legs
```

Comment:
This makes all inhabitants to have paws AND hands AND legs.
Some of those propwerties are just ignore in some cases, but still all inhabitants have those.
Of the points of the OOP si that any predecessor has only properties that are commong for its ancestors, and every child (ancestor) class introduces properties unique to it (and shared by its ancestors if any).